spec/spec_helper_acceptance.rb

```diff
@@ -145,9 +163,20 @@ def teardown_puppet_on(host)
   # Note pc1_repo is specific to the module's manifests. This is knowledge we need to clean
   # the machine after each run.
   pp = <<-EOS
-package { ['puppet-agent', 'puppet', 'mcollective', 'mcollective-client', 'activemq']: ensure => purged }
-file { ['/etc/puppet', '/etc/puppetlabs', '/etc/mcollective', '/etc/activemq']: ensure => absent, force => true, backup => false }
-yumrepo { 'pc1_repo': ensure => absent }
+package { ['puppet-agent', 'puppet', 'mcollective', 'mcollective-client']: ensure => purged }
+file { ['/etc/puppet', '/etc/puppetlabs', '/etc/mcollective']: ensure => absent, force => true, backup => false }
+#{
+  case host['platform']
+  when /debian|ubuntu/
+    on host, '/opt/puppetlabs/bin/puppet module install puppetlabs-apt', { :acceptable_exit_codes => [0,1] }
+    "include apt\napt::source { 'pc1_repo': ensure => absent }"
```

Comment:
@MikaelSmith seeing a strange error when puppet_agent tries to purge the deb repo configs: https://gist.github.com/joshcooper/0be2740f2a1be0b194d4#file-gistfile1-txt so I think state is getting left behind?